### PR TITLE
Upgrade http transport version and code refactoring

### PR DESCRIPTION
**Pull request by Yasara123**

### CODE REVIEW

File: component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpSource.java [hunk 1]
```diff
@@ -322,18 +323,25 @@
                                 "`Authorization:'Basic encodeBase64(username:Password)'` header.")},
         systemParameter = {
                 @SystemParameter(
-                        name = "serverBootstrapBossGroupSize",
+                        name = "SERVER_BOOTSTRAP_BOSS_GROUP_SIZE1",
```
Comment (pcnfernando) [inline component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpSource.java]: Is "SERVER_BOOTSTRAP_BOSS_GROUP_SIZE1" name intentional

File: component/src/main/java/org/wso2/extension/siddhi/io/http/sink/HttpSink.java [hunk 2]
```diff
@@ -719,15 +708,17 @@ private HTTPCarbonMessage generateCarbonMessage(List<Header> headers, String con
...
-        cMessage.setProperty(Constants.HOST, httpURLProperties.get(HttpConstants.HOST));
+        cMessage.setProperty(Constants.HTTP_HOST, httpURLProperties.get(Constants.HTTP_HOST));
         //set port
-        cMessage.setProperty(Constants.PORT, Integer.valueOf(httpURLProperties.get(HttpConstants.PORT)));
+        cMessage.setProperty(Constants.HTTP_PORT, Integer.valueOf(httpURLProperties.get(Constants.HTTP_PORT)));
         // Set method
-        cMessage.setProperty(HttpConstants.HTTP_METHOD, httpMethod);
+        cMessage.setProperty(Constants.HTTP_METHOD, httpMethod);
+        //cMessage.setProperty(Constants.HTTP_METHOD, Constants.HTTP_GET_METHOD);
+        //cMessage.setProperty(Constants.REQUEST_URL, httpURLProperties.get(Constants.REQUEST_URL));
```
Comment (Anoukh) [inline component/src/main/java/org/wso2/extension/siddhi/io/http/sink/HttpSink.java]: Can we remove these comments?

File: component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpConnectorPortBindingListener.java [hunk 3]
```diff
@@ -30,41 +29,41 @@
...
         }
         connectorStartupSynchronizer.getCountDownLatch().countDown();
     }
-
+    
     @Override
     public void onClose(String serverConnectorId, boolean isHttps) {
         if (isHttps) {
-            log.info("siddhi: stopped HTTPS server connector " + serverConnectorId);
+            log.info("HTTPS source " + serverConnectorId + " has been started");
```
Comment (Anoukh) [inline component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpConnectorPortBindingListener.java]: Should this be has been closed?

File: component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpConnectorPortBindingListener.java [hunk 4]
```diff
@@ -30,41 +29,41 @@
...
         } else {
-            log.info("siddhi: stopped HTTP server connector " + serverConnectorId);
+            log.info("HTTP source " + serverConnectorId + " has been started");
```
Comment (Anoukh) [inline component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpConnectorPortBindingListener.java]: Same comment as above

Comment (Yasara123) [reply component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpConnectorPortBindingListener.java]: Fixed in f606aca

File: component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpConnectorPortBindingListener.java (hunk 3 above)
Comment (Yasara123) [reply component/src/main/java/org/wso2/extension/siddhi/io/http/source/HttpConnectorPortBindingListener.java]: Fixed in f606aca

File: component/src/main/java/org/wso2/extension/siddhi/io/http/sink/HttpSink.java (hunk 2 above)
Comment (Yasara123) [reply component/src/main/java/org/wso2/extension/siddhi/io/http/sink/HttpSink.java]: Fixed in f606aca

**Pull request merged**